region/brinstar/red/Red Tower.json

```
@@ -340,7 +340,7 @@
       ]
     },
     {
-      "name": "Red Tower Spark",
+      "name": "Red Tower Hero Shot Spark",
```

I'm preferring moving towards full long-hand names, but I'm fine either way.
```suggestion
      "name": "Red Tower Hero Shot Shinespark",
```